.travis.yml

```diff
@@ -12,10 +12,12 @@ julia:
   - nightly
 
 notifications:
-    email: false
+  webhooks: https://coveralls.io/webhook
+  email: false
 
 script:
   - julia --check-bounds=yes etc/travis-test.jl
+  - export COVERALLS_PARALLEL=true
```

Comment:
Not a reason to hold up this PR, just wondering, but: is there any reason to do it like this, and now how the README suggests to do i? I.e.,
```
env:
  global:
    - COVERALLS_PARALLEL=true
```

Reply:
Just that it confuses testing to set it globally. We similarly don't use the suggested script either.